content/terraform/v1.13.x/docs/cli/commands/output.mdx

```diff
@@ -92,6 +92,20 @@ instance_ips = [
 ]
 ```
 
+The `terraform output` command only displays outputs from the root module. 
+To display outputs from child modules, you must define a new output in the 
+root module with a value of the child submodule output.
+
+```hcl
+module "random" {
+  source = "./modules/random"
+}
+
+output "string" {
```

Comment:
I'm sorry to be that guy, but can we make this a little more realistic? Like make it an IP address or something?